samples/client/petstore/typescript-node/default/.openapi-generator/FILES

```diff
@@ -5,10 +5,10 @@ api/petApi.ts
 api/storeApi.ts
 api/userApi.ts
 git_push.sh
-model/apiResponse.ts
-model/category.ts
-model/order.ts
-model/pet.ts
-model/tag.ts
-model/user.ts
+model/./apiResponse.ts
```

Comment:
Why this change? ./ makes not much sense in my opinion.

Reply:
Agreed `./` is not needed. We'll tackle it separately.

Reply:
Ok, so there is nothing left here to todo for the PR?